rails-questions.md

```diff
@@ -2,22 +2,48 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
+To create a new rails app I would run 
 
+rails new app_name -d  postgresql -T
+
+ceating the new app with my name in snake case then deleting the native database and replacing it with postgresql the -T would remove the native testing suite
 
 2. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
 
+I would want to create a new column to hold the foreign id using a migration 
+
+I would use command 
+rails g migration add_childtablename_id_to_parenttablename 
+
+then I would go to the text editor and in the migeration tab with the most recent time stamp and edit the change
+
+def change
+change-table :Parenttableaname Do |t|
+        T.integer:childtablename
+        end
+    end
+end
+
+Then I would head back to the terminal and run comand
+
+rails db:migrate 
+
 
 
 3. What is the command to generate a Rails model for a person table with columns named shirt, pants, and shoes?
 
+To generate a Rails table with colums named :shirt, :pants, :shoes I would run command generate or g followd by the name of the the table, which would have to be different from the name of the database
 
+rails g model Wardrobe shirt: integer, pants: integer, shoes: integer
```

Comment:
```suggestion
rails g model Wardrobe shirt:integer pants:integer shoes:integer
```

The extra spaces between the column_name and data type are going to cause issues and generate unintended columns. Also I believe commas are not needed (🤔 )